styles/button.css

```diff
@@ -0,0 +1,228 @@
+    /* Button Starting Here */
+    .btn {
+        padding: 0.7rem;
+        color: #fff;
+        max-width:max-content;
+        border-radius: 0.3rem;
+        margin: 0 1rem 0 0;
+        cursor: pointer;
+    }
+    .btn a {
+        all: unset;
+    }
+    /* Primary Button */ 
+    .btn-primary {
+        border: 0.2rem solid var(--primary-color);
+        background: var(--primary-color);
+        
+    }
+    /* secondory Button */ 
+    .btn-secondory {
+        border: 0.2rem solid var(--secondory-text-color);
+        background: var(--secondory-text-color);
+    }
+    /* success Button */ 
+    .btn-success {
+        border: 0.2rem solid var(--successColor);
+        background: var(--successColor);
+    }
+    /* warning Button */ 
+    .btn-warning {
+        border: 0.2rem solid var(--warningColor);
+        background: var(--warningColor);
+        color: var(--primary-text-color);
+    }
+    /* danger Button */ 
+    .btn-danger {
+        border: 0.2rem solid var(--dangerColor);
+        background: var(--dangerColor);
+    }
+    /* dark Button */ 
+    .btn-dark {
+        border: 0.2rem solid var(--darkColor);
+        background: var(--darkColor);
+        
+    }
+    /* light Button */ 
+    .btn-light {
+        border: 0.2rem solid var(--lightColor);
+        background: var(--lightColor);
+        color: var(--primary-text-color);
+    }
+    /* information Button */ 
+    .btn-info {
+        border: 0.2rem solid var(--infoColor);
```

Comment:
Use a consistent naming convention. You've used hyphenated words (eg. `--primary-color`) but here camel case (`--infoColor`)